lightstep-tracer-jre/build.gradle

```diff
@@ -43,10 +43,14 @@ test {
 // Defines the repositories need for dependencies below
 repositories {
     jcenter()
+    maven {
+        url  "http://oss.jfrog.org/oss-snapshot-local/"
```

Comment:
maybe add a comment explaining that this is where the OpenTracing beta artifacts are published?